includes/class-edd-cli.php

```diff
@@ -1201,6 +1203,9 @@ public function migrate_customer_data( $args, $assoc_args ) {
 		if ( ! $force && $customer_email_addresses_complete ) {
 			WP_CLI::warning( __( 'The user email addresses custom table migration has already been run. To do this anyway, use the --force argument.', 'easy-digital-downloads' ) );
 		} else {
+
+			WP_CLI::line( __( 'Prepairing to migrate customer email addresses (this can take several minutes).', 'easy-digital-downloads' ) );
```

Comment:
```suggestion
			WP_CLI::line( __( 'Preparing to migrate customer email addresses (this can take several minutes).', 'easy-digital-downloads' ) );
```